src/runtime/runtime.cs

```diff
@@ -70,30 +126,69 @@ public static IntPtr GetProcAddress(IntPtr dllHandle, string name)
             }
             return res;
         }
+        
+        public static IntPtr dlopen(String fileName, int flags)
```

Comment:
It looks like a Windows application could call dl{open,sym,close,...}. I saw the code above never does it on Windows but should you throw an exception if the platform is windows?